README.md

```diff
@@ -1,115 +1,115 @@
-# Torch-TensorRT
+<div align="center">
+
+Torch-TensorRT
+===========================
+<h4> Easily achieve the best inference performance for any PyTorch model on the NVIDIA platform. </h4>
 
 [![Documentation](https://img.shields.io/badge/docs-master-brightgreen)](https://nvidia.github.io/Torch-TensorRT/)
+[![pytorch](https://img.shields.io/badge/PyTorch-2.2-green)](https://www.python.org/downloads/release/python-31013/)
+[![cuda](https://img.shields.io/badge/cuda-12.1-green)](https://developer.nvidia.com/cuda-downloads)
+[![trt](https://img.shields.io/badge/TensorRT-8.6.1-green)](https://github.com/nvidia/tensorrt-llm)
+[![license](https://img.shields.io/badge/license-BSD--3--Clause-blue)](./LICENSE)
 [![CircleCI](https://circleci.com/gh/pytorch/TensorRT.svg?style=svg)](https://app.circleci.com/pipelines/github/pytorch/TensorRT)
```

Comment:
Can you replace this with the GHA version, otherwise it will always say failing

Reply:
I'm not super familiar with this, so bear with me...

I think you need to pick a workflow to create a badge, and it seems like "Build and test Linux wheels" and "Build and test Windows wheels" would both be relevant -- Do you agree?

I think it'd be better if there were a single badge representing all workflows -- is this possible?

Here's the badge URL I was looking at: https://github.com/pytorch/TensorRT/actions/workflows/build-test-linux.yml/badge.svg

Reply:
I think having both is ok because there are workflows other than linux and windows (Docs, Docker, some issue management stuff) that might make it confusing